PROJECT

```diff
@@ -3,7 +3,7 @@ layout:
 - go.kubebuilder.io/v2
 plugins:
   go.sdk.operatorframework.io/v2-alpha: {}
-projectName: kubernetes-power-manager
+projectName: intel-kubernetes-power-manager
```

Comment:
Just curious about why the change in project name? It seems possible that kubernetes-power-manager could be made to work with other processor architectures in the future.

Reply:
If I remember correctly it was done to keep the project name in sync with the name of the operator in the RH catalog.